### PR TITLE
Add neo4j-admin-import section and parameter details for Parquet.

### PR DESCRIPTION
**Pull request by meistermeier**
The Parquet file support for neo4j admin import will come out in on of the next minor versions as a preview feature.
Depending on the feedback we get from customers and users, there will be definitely coming more (also to the docs).
This is a quite defensive change to avoid promising too much but also pointing out that this feature exists at all ;)

Because the feature itself is not merged yet, I added the _DO NOT MERGE_ label. 
Please let us get this into a shape where we can _just_ merge it after the feature went into the product, thanks.

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1850.surge.sh

Updated pages:
- [Import](https://neo4j-docs-operations-1850.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1850.surge.sh

Updated pages:
- [Import](https://neo4j-docs-operations-1850.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by fbiville** [comment]
Just to be extra explicit: the feature has been merged so this PR can be reviewed/merged too.

File: modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc [hunk 1]
```diff
@@ -25,6 +25,17 @@ Change Data Capture does **not** capture any data changes resulting from the use
...
 ====
 
+[role=label--beta]
+== Parquet file support
+Starting with Neo4j 5.24, Neo4j provides support for the Parquet file format in a public beta version.
+The additional parameter `--input-type [csv|parquet]` was introduced to explicitly tell the importer to use either CSV or Parquet.
+Its value defaults to CSV if it is not defined.
+
+Most of the parameters that can be used to configure the import are also valid for the Parquet format.
+There are indicators in the parameter overview to point out which parameters are supported.
```
Comment (fbiville) [inline modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc:35]:
```suggestion
There are indicators in the parameter overview to point out which parameters are not supported.
```
It seems the indicators below are for parameters that the Parquet importer does not support/ignores, or?

**Comment by renetapopova** [comment]
> Just to be extra explicit: the feature has been merged so this PR can be reviewed/merged too.

I am reviewing the PR now, but I see that @meistermeier is on vacation. So, I'll just push my editorial and style changes into the PR. We can always discuss and improve them later when he is back.

**Comment by renetapopova** [comment]
Also, I need to see why the test is failing.

File: modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc (hunk 1 above)
Comment (renetapopova) [reply modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc:35]: Yes, I think you're right.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1850.surge.sh

Updated pages:
- [Import](https://neo4j-docs-operations-1850.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Review by renetapopova — COMMENTED** [review]
I think it looks good now, but we need to make sure the test is green before merging it.

**Comment by fbiville** [comment]
@renetapopova I'm covering for @meistermeier. I can work on any updates you deem necessary.

**Comment by renetapopova** [comment]
> @renetapopova I'm covering for @meistermeier. I can work on any updates you deem necessary.

Ah, cool! As I said, I pushed some editorial updates if you want to take a look, but I think the content looks good for merging. However, the `test neo4j-admin tools (PRs) (Testing)—TeamCity build` failed, so I'll wait for a few other PRs to be merged first, rebase this one, and then investigate further if the test is still failing.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1850.surge.sh

Updated pages:
- [Import](https://neo4j-docs-operations-1850.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1850.surge.sh

Updated pages:
- [Import](https://neo4j-docs-operations-1850.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1850.surge.sh

Updated pages:
- [Import](https://neo4j-docs-operations-1850.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1850.surge.sh

Updated pages:
- [Import](https://neo4j-docs-operations-1850.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by renetapopova** [comment]
Closing this one as duplicate.

**Pull request closed without merging**